src/Geometry.cpp

```diff
@@ -1151,6 +1161,16 @@ void Geometry::setFSRsToKeys(std::vector<std::string>* FSRs_to_keys) {
 
 
 /**
+ * @brief Sets the method for selection of the maximum ring radius for subdivision
+ * @param ringify_type The method to use for selection of maximum ring radius
+ */
+void Geometry::setRingifyType(ringifyType ringify_type)
+{
```

Comment:
Per the [google c++ style guide](https://google.github.io/styleguide/cppguide.html#Function_Declarations_and_Definitions), the open curly brace is always on the end of the last line of the function declaration, not the start of the next line.